samples/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -233,6 +247,41 @@ static bool event_handler(const struct event_header *eh)
 		return false;
 	}
 
+	if (IS_ENABLED(CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE)) {
+		if (is_config_forward_event(eh)) {
+			struct config_forward_event *event = cast_config_forward_event(eh);
+
+			if (!bt_gatt_hids_c_ready_check(&hidc)) {
+				LOG_WRN("Cannot forward, peer disconnected");
+				return false;
+			}
+
+			struct bt_gatt_hids_c_rep_info *config_rep =
+				bt_gatt_hids_c_rep_find(&hidc,
+					BT_GATT_HIDS_C_REPORT_TYPE_FEATURE,
+					REPORT_ID_USER_CONFIG);
+			if (config_rep == NULL) {
+				LOG_ERR("Feature report not found");
+				return false;
+			}
+
+			u8_t data[REPORT_SIZE_USER_CONFIG];
+
+			memcpy(&(data[0]), &event->recipient, sizeof(event->recipient));
+			data[2] = event->id;
```

Comment:
what about endianess?

Reply:
What do you mean? These reports are received as little endian.